redhat-operators/kie-cloud-operator/kiecloud-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,210 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: kiecloud-operator.v0.1.0
+  namespace: placeholder
+  annotations:
+    categories: A list of comma separated categories that your operator falls under.
+    certified: The operator's certification. This field should be set to false until a process to get this value is defined at a later date.
+    description: KieCloud can deploy RHPAM environments in the form of KieApp objects.
+    containerImage: quay.io/kiegroup/kie-cloud-operator:v0.1.0
+    createdAt: The date that the operator was created. The format should match yyyy-mm-ddThh:mm:ssZ
+    support: Red Hat, Inc.
+    tectonic-visibility: ocs
+    alm-examples: >-
+      [{"apiVersion":"app.kiegroup.org/v1","kind":"KieApp","metadata":{"name":"trial"},"spec":{"environment":"trial"}}]
+  labels:
+    operator-kiecloud: "true"
+spec:
+  displayName: KieCloud
+  description: KieCloud can deploy RHPAM environments in the form of KieApp objects.
+  keywords: [kieapp rhpam kie cloud pam kiecloud]
+  version: 0.1.0
+  maturity: alpha
```

Comment:
This might conflict with the channel set in `package.yaml`

Reply:
changed to "beta"